api/routes/model.js

```diff
@@ -0,0 +1,198 @@
+const express = require("express");
+const router = express.Router();
+const validate = require("express-jsonschema").validate;
+
+const ModelSchema = {
+  id: "/ModelSchema",
+  type: "object",
+  properties: {
+    name: {
+      type: "string",
+      required: true
+    },
+    sdg: {
+      type: "array",
+      items: {
+        type: "integer"
+      },
+      required: true
+    },
+    description: {
+      type: "string",
+      required: true
+    },
+    stages: {
+      type: "object",
+      patternProperties: {
+        ".*": {
+          type: "object",
+          properties: {
+            stakeholders: {
```

Comment:
for both stakeholder, challenges, and insights, can we have the contents have a name and a description rather than just being strings